src/Compilers/Core/Portable/SpecialMember.cs

```diff
@@ -194,6 +194,8 @@ internal enum SpecialMember
 
         System_Type__GetTypeFromHandle,
 
+        System_String__SubstringInt,
```

Comment:
Please put this with the other `String` members. Also, consider renaming the existing `Substring` member to include the parameter types to make it clear what the difference is.

Reply:
I thought it's not allowed to change the order or rename existing ones. 
Thank you for allowing them.

Reply:
Is this comment applied to `WellKnownMember(s)` too?

Reply:
Changing `WellKnownMembers` is a bit more complex due to its size, and various offset flags that go into parts of decoding it. I'd just add to the end of that one. `SpecialType` is not fine to change, since it's a public type, but `SpecialMember` is not a public type so it can be changed.

Reply:
I see. I'll change only `SpecialMember`.